spark2.4/build.sbt

```diff
@@ -7,7 +7,7 @@ scalaVersion := "2.11.12"
 
 //val spliceVersion = "2.8.0.1945"
 //val spliceVersion = "3.1.0.1967-SNAPSHOT"
-val spliceVersion = "3.2.0.1980-SNAPSHOT"
+val spliceVersion = "3.2.0.1985-SNAPSHOT"
```

Comment:
While you're at it, why not to remove the above commented-out versions too?

Reply:
Removed in commit https://github.com/splicemachine/splice-machine-spark-connector/pull/27/commits/f67f17908f97ceeaddf328c82576ad815f7e3afa